### PR TITLE
Prepare for real data

### PR DESCRIPTION
**Pull request by aJerryGuo**
## Summary

1. Refactor `seeds.exs`
2. Use script to generate data that can be inserted into `seeds.exs`

### CODE REVIEW

**Review by aStanleyLiang — APPROVED** [review]
LGTM

File: lucky_draw/scripts/candidate_example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+You Know Who
+Queen
+King
```
Comment (aChinKaiWu) [inline lucky_draw/scripts/candidate_example:3]: One moment, I'm preparing the list.

Comment (aChinKaiWu) [reply lucky_draw/scripts/candidate_example:3]:
```
Eric Liao
Giles Shao
Hank Lin
Henry Huang
Jerry Guo
Joy Chiu
King Yang
Michael Poernomo
Mike Chen
Robi Hamanto
Rufus Tsai
Samantha Huang
Stanley Liang
Steven Chiu
Wilson Chen
Yishin Ho
Jim Lin
Leon Chang
Peter Tsai
Rex Yeh
Gilles Bertal
Edward Huang
Kingsley Lim
Vincent Yang
Cindy Lu
Mark Huang
諸岡さん
```

Rules
1. 社員限定（管理者除外）
2. 不在現場者視同放棄

Comment (aStanleyLiang) [reply lucky_draw/scripts/candidate_example:3]: Ok, I will handle it

Comment (aChinKaiWu) [reply lucky_draw/scripts/candidate_example:3]: Considering the privacy, I will send the list privately.
The change LGTM.

**Pull request merged**